lib/auth/spark/model.rb

```diff
@@ -0,0 +1,8 @@
+module Devise
+  module Models
+    module SparkAuthenticatable
+      extend ActiveSupport::Concern
+
```

Comment:
This is the place to get/set information from the Spark response. Should probably do that.